iothub/service/src/IotHubClientWebSocket.cs

```diff
@@ -229,24 +229,19 @@ public async Task<int> ReceiveAsync(byte[] buffer, int offset, int size, TimeSpa
                 {
                     // Ignore pong frame and start over
                     totalBytesRead = 0;
-                    do
```

Comment:
This while loop is unnecessary as totalBytesRead always resets back to 0 and we only read 2 bytes from the stream.